testing/features/support/helpers/env_variables.rb

```diff
@@ -55,7 +55,11 @@ def headless?
     end
 
     def base_url
-      ENV.fetch("APP_URL", "https://citizensadvice.github.io/design-system")
+      if ENV["APP_ENVIRONMENT"] == "local"
```

Comment:
can we refactor this and include a secondary method.

So lets have
`def environment` that takes the input, does a fallback to sym - Make the fallback the master website.
then `base_url` takes the symbol and does the if/else

Reply:
changes applied and  pushed @luke-hill